src/measure_theory/interval_integral.lean

```diff
@@ -389,23 +389,23 @@ begin
   { simp [(show 0 < c, from h), mul_div_cancel, hc, abs_of_pos] }
 end
 
-@[simp] lemma integral_comp_mul_right' (c) :
+lemma smul_integral_comp_mul_right (c) :
```

Comment:
Why do you remove the `simp` here?

Reply:
Is that indeed useful? I can't think of an instance (within the constraints of what is currently in mathlib) where I would want to compute the integral of a function that maps to something other than `ℝ`.

Reply:
What about the complex numbers? Or can mathlib not do that yet?

Reply:
Put more simply, I can't see leaving this lemma marked `simp` as being harmful, unless the linter is telling you that it is.

Reply:
As far as I am aware that is not yet possible in the same manner it is for reals. But I can add the `simp`s back, no problem. There shouldn't be a problem with the linter since the new lemmas are in a lower file.

Reply:
I restored the old `simp`s. I also added `simp` to `integral_smul` for the same reason.